### ISSUE TITLE
[BUG]Can't extend and collapse special button menu on CompanionPi

### ISSUE THREAD

**Issue by LHEGDIS** [bitfocus/companion#3077]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Since I installed CompanionPi V3.4.1+249 on the Raspberry Pi 5 I am not able to extend the special button menu when creating a new button. This means I cant create a button to switch through the pages, nor can I createa button that displays what page I'm on.

### Steps To Reproduce

1. Go to "Buttons".
2. Select an empty buton.
3. Select the arrow button on the right next to the button "create button".
4. Nothing happens.

### Expected Behavior

It should extend a menu with the Options "Regular button", "Page up", "Page number" and "Page down".

### Environment (please complete the following information)

```markdown
- OS: CompanionPi (for the Browser: Windows 11)
- Browser: Firefox
- Companion Version: CompanionPi V3.4.1+249
```

**Comment by knelsonkc** [comment]
Same issue on fresh install of v3.4.1+249

**Comment by Julusian** [comment]
Perfect timing in reporting this, this is now fixed in the v3.4.2 release
